### ISSUE TITLE
Add option to customize back reference aria-label text

### ISSUE THREAD

**Issue by SebastianKohler** [bent10/marked-extensions#171]
- Extension Name: marked-footnote
- Extension Version: 1.2.4

### Use case

Markdown texts with footnotes in non-English languages.

### Proposal

Add an option to customize the aria-label text of footnote back references. Currently the aria-labels are hard coded as `Back to reference #`. The "Back to reference" part should be customizable, so the back reference links don't have aria-labels in English if the text is otherwise in another language.

Being able to customize `description` is already a great feature!

**Comment by SebastianKohler** [comment]
Great, thank you! Any estimate on when the an updated package will be released?

**Comment by bent10** [comment]
I've just released the new version. Please check it out 🚀

Thanks!
